addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -226,3 +226,19 @@ body.editor_enable {
         }
     }
 }
+
+// Website Animate
+.editor_enable.o_animated_text_highlighted {
+    .o_animated_text {
+        position: relative;
+
+        &:after {
+            content: "";
+            pointer-events: none;
+            @include o-position-absolute(0, 0, 0, 0);
+            z-index: 1;
+            border: 1px dotted white;
```

Comment:
this border is usefull when there is a green background

Reply:
Add that as a comment in the code, not on the PR :wink: